components/molecule/avatar/src/index.js

```diff
@@ -11,7 +11,7 @@ import AvatarBadge, {
   AVATAR_BADGE_SIZES
 } from './AvatarBadge'
 
-export const AVATAR_SIZES = {
+const AVATAR_SIZES = {
```

Comment:
export too, 4 retrocompatibility

Reply:
Yes, is exported at bottom: https://github.com/SUI-Components/sui-components/pull/1359/files#diff-de7b3615c3bc225b5005541eaf5929362ec3e7c6d21d9fd139842817d0e56bf5R91

Reply:
Done @andresin87 https://github.com/SUI-Components/sui-components/pull/1359/commits/32d6c888be04d9377d8a57cfa081cbdaec4f94b5